src/python/WMCore/MicroService/Unified/Transferor.py

```diff
@@ -168,37 +384,56 @@ def add(self, requests):
                 datasets = rdict.get('datasets')
                 sites = rdict.get('sites')
                 subscription = PhEDExSubscription(datasets, sites, self.group)
-                if self.verbose:
-                    print("### add subscription", subscription)
+                self.logger.debug("### add subscription %s" % subscription)
                 # TODO: when ready enable submit subscription step
                 # self.phedex.subscribe(subscription)
 
+		# TODO: when we made subscription we change status of request in ReqMgr2
+
     def info(self, request=None):
```

Comment:
Shouldn't `request` parameter be mandatory here? If it's not provided, what exactly is it returning?

Reply:
ok